peepholelib/models/model_wrap.py

```diff
@@ -66,6 +66,7 @@ def __init__(self, **kwargs):
         
         # set in set_model()
         self._model = None
+        self.num_classes = None
```

Comment:
It seems like we are assuming that the model does not do the `softmax` and returns an `int` corresponding to a class. We gotta keep an eye on this.